docs/resources/samples/clear-table-filter-for-active-cell.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Clear table column filter based on active cell location in Office Scripts'
+description: 'Learn how to clear table column filter based on active cell location.'
+ms.date: 03/02/2021
+localization_priority: Normal
+---
+
+# Clear table column filter based on active cell location
+
+Shows how to clear the table column filter based on active cell location. The script detects if the cell is part of a table and then determines the table column and clears any filter that may be applied on it.
+
+If you wish to learn more about how to save the filter prior to clearing it (and re-apply later), check out a [more advanced sample](move-rows-across-tables.md).
+
+## Video
```

Comment:
How do we want to order these parts? Right now, it's:
- Video
- Images
- Script

Does that seem like the right approach? Or should we give code first? My assumption is people want the sample up front (particularly for smaller, more self-evident samples), but I could go either way.
- Script
- Image
- Video

Reply:
Right now I'm leaning toward:
- Image
- Script
- Video

Reply:
Works for me!